client/sites/components/sites-dataviews/index.tsx

```diff
@@ -119,16 +122,32 @@ const DotcomSitesDataViews = ( {
 	const fields = useMemo< Field< SiteExcerptData >[] >(
 		() => [
 			{
-				id: 'site',
+				id: 'title-text',
 				label: __( 'Site' ),
-				header: <span>{ __( 'Site' ) }</span>,
-				getValue: ( { item }: { item: SiteExcerptData } ) => item.URL,
+				header: <span></span>,
+				getValue: ( { item }: { item: SiteExcerptData } ) => item.title,
 				render: ( { item }: { item: SiteExcerptData } ) => {
 					return <SiteField site={ item } openSitePreviewPane={ openSitePreviewPane } />;
 				},
 				enableHiding: false,
 				enableSorting: true,
 			},
+			{
+				id: 'site-url',
```

Comment:
For sorting, the value of this field will be `item[ 'site-url' ]`. Does this value exist in the data source? You can provide a getValue function ([docs](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dataviews/#getvalue)) if the field id and the value's object key differ.

Reply:
Thanks, I was wondering how sorting fits into this.